infra/scripts/atlas_db_init.sh

```diff
@@ -8,14 +8,51 @@ LOG_FILE=${AZ_SCRIPTS_PATH_OUTPUT_DIRECTORY}/all.log
 exec >  >(tee -ia "${LOG_FILE}")
 exec 2> >(tee -ia "${LOG_FILE}" >&2)
 
-apk --update add postgresql-client gettext
-admin_user_password="${OHDSI_ADMIN_PASSWORD}${OHDSI_ADMIN_USERNAME}"
-app_user_password="${OHDSI_APP_PASSWORD}${OHDSI_APP_USERNAME}"
-admin_md5=$(echo -n "$admin_user_password" | md5sum | awk '{ print $1 }')
-export admin_md5="'md5$admin_md5'"
+apk --update add postgresql-client gettext python3 py3-pip
 
-app_md5=$(echo -n "$app_user_password" | md5sum | awk '{ print $1 }')
-export app_md5="'md5$app_md5'"
+
+# Embedded Python script to generate SCRAM-SHA-256 hashes
+generate_scram_hash() {
+    local password=$1
+    python3 - <<END
+from base64 import standard_b64encode
+from hashlib import pbkdf2_hmac, sha256
+from os import urandom
+import hmac
+import sys
+
+
+salt_size = 16
+digest_len = 32
+iterations = 4096
+
+
+def b64enc(b: bytes) -> str:
+    return standard_b64encode(b).decode('utf8')
+
+
+def pg_scram_sha256(passwd: str) -> str:
+    salt = urandom(salt_size)
+    digest_key = pbkdf2_hmac('sha256', passwd.encode('utf8'), salt, iterations,
+                             digest_len)
+    client_key = hmac.digest(digest_key, 'Client Key'.encode('utf8'), 'sha256')
+    stored_key = sha256(client_key).digest()
+    server_key = hmac.digest(digest_key, 'Server Key'.encode('utf8'), 'sha256')
+    return (
+        f'SCRAM-SHA-256\${iterations}:{b64enc(salt)}'
+        f'\${b64enc(stored_key)}:{b64enc(server_key)}'
+    )
+if __name__ == "__main__":
+    password = "$password"
+    print(pg_scram_sha256(password))
```

Comment:
is printing the hashed password to stdout necessary? that could be logged and leaked I think

Reply:
I agree it's not ideal, however in the usage itself it's not being printed to stdout rather than put into a var. I couldnt find a more elegant way to combine python and bash together...

Reply:
I get it, user need to pip install anything before running it?

Reply:
Pip not needed here, can work without it